internal/controllers/port/tests/port-create-full/00-port.yaml

```diff
@@ -65,4 +65,6 @@ spec:
     - port-create-full
     addresses:
     - subnetRef: port-create-full
-      ip: 192.168.155.122
\ No newline at end of file
+      ip: 192.168.155.122
+    vnicType: normal
```

Comment:
`vnicType: normal` and `portSecurityEnabled: true` are the default values, so we're not really testing anything. It would be more interesting to set these to different values, and see if the change is reflected. But I see that `portSecurityEnabled: false` is not compatible with security groups or allowed address pairs, so we can't set these values in the `create-full` test. We need to create a new test called `port-create-sriov` or something like that.

We would also want to check the default values in the `create-minimal` test assert.

Reply:
yeah, needs more e2e tests of this one, will do :+1:

Reply:
@itzikb-redhat this is still missing a `port-create-sriov` test where we test both `portSecurityEnabled: false` and `vnicType: direct` at the same time. I think it would be a useful addition to validate the sr-iov scenario.